velox/docs/functions/binary.rst

```diff
@@ -22,6 +22,14 @@ Binary Functions
 
     Computes the SHA-512 hash of ``binary``.
 
+.. function:: spooky_hash_v2_32(binary) -> varbinary
```

Comment:
In the documentation function should appear in alphabetical order. Please, move these accordingly.

Reply:
@mbasmanova : There are other more valid inconsistencies in this file in the alphabetical order : hmac_sha() functions need to be moved before md5 and xxhash64 needs to be moved later in the file. In a previous review comment I asked @isadikov to fix the ordering in general in a subsequent PR. Hope that is okay.

Reply:
I see. It is fine to fix existing problems in the separate PR, but let's not introduce more problems with the new functions.

Reply:
This PR doesn't introduce any new problems as the order of these functions is correct relative to the sha... functions above it. Its the rest of the orders that need to be fixed for the inconsistency.

Reply:
Got it. Thank you for clarifying.